flight/Modules/Actuator/actuator.c

```diff
@@ -295,10 +294,30 @@ static void actuator_task(void* parameters)
 			if (flightStatus.FlightMode != FLIGHTSTATUS_FLIGHTMODE_FAILSAFE) {
 				throttle_source = manual_control_command.Throttle;
 			}
+		} else {
+			throttle_source = desired.Thrust;
 		}
-		else throttle_source = desired.Thrust;
 
-		bool positiveThrottle = throttle_source >= 0.00f;
+		bool stabilize_now = throttle_source >= 0.0f;
+
+		static uint32_t last_pos_throttle_time = 0;
+
+		if (stabilize_now) {
+			if (actuatorSettings.LowPowerStabilizationMaxTime) {
+				last_pos_throttle_time = this_systime;
+			}
+
+			// Could consider stabilizing on a positive arming edge,
+			// but this seems problematic.
+		} else if (last_pos_throttle_time) {
```

Comment:
I believe that will happen without this changeset, too.  In either case there's clip.  Perhaps open a bug?  I need to improve the actuator alarms in general before release because they're too chatty.